File: expressions/budget.py
def enlightenMe(type):
  enter = input(f"what is your monthly {type}: ")
  return int(enter)



print("This low level calculator made by some guy who needs a good grade and will kinda calculate your expenses.")
inIncome= enlightenMe("income")
rent= enlightenMe("rent")
utilities= enlightenMe("utilites")
Groceries= enlightenMe("groceries")
transportation= enlightenMe("transportation")


def calculate(type,amount):
 per = amount/inIncome * 100
 print(f"your {type} is {per}% of your income. ")

expenses = inIncome-rent-utilities-Groceries-transportation
savings= inIncome*.2
spending = inIncome-expenses

print("Your monthly savings is ",savings)
print("Your monthly expenses is ", expenses)
print("Your spending money is",spending)
 
calculate("rent",rent)
calculate("utilities",utilities)
calculate("groceries",Groceries)
calculate("transportation",transportation)
#put it on the beat bro

File: expressions/budget.c
#include <stdio.h>

int main(void){
   float income, rent, utilities, groceries, transportation, expenses, savings, total; 
   float prent, putilities, pgroceries, ptransportation, pexpenses;
   printf ("this is going to calculate your budget for the month.");
   printf ("how much do you make a month?\n: ");
   scanf("%f", &income); //what
  printf ("how much is rent?\n ");
 scanf("%f", &rent);
  printf ("how mucch is utilites?\n ");
scanf("%f", & utilities);
 printf ("how mucch is groceries?\n ");
scanf("%f", & groceries);
printf ("how mucch is transportation?\n ");
scanf("%f", & transportation);

expenses = rent + utilities + groceries + transportation;
total = income - expenses - savings;
savings = income*.2;
prent = rent/income*100;
putilities = utilities/income*100;
pgroceries = groceries/income*100;
ptransportation = transportation/income*100;
pexpenses = expenses/income*100;

   printf("your income is: $%.2f\n", income);
   printf("your expenses is: $%.2f\n", expenses);
   printf("your savings is: $%.2f\n", savings);
   printf("your total left to spend is: $%.2f\n", total);
printf("your rent is %.2f%% of you income", "%d",prent);

    return 0;
}